provider.tf

```
@@ -7,8 +7,16 @@ terraform {
   }
 
   required_version = ">=1.2.0"
+
+  backend "s3" {
+    bucket = "terraform-rem-state-stor"
+    key = "terraform.tfstate"    
+    region = "eu-central-1"
+    dynamodb_table = "remote-state"
```

```suggestion
    bucket         = "terraform-rem-state-stor"
    key            = "terraform.tfstate"    
    region         = "eu-central-1"
    dynamodb_table = "remote-state"
```

you can also run `terraform fmt -recursive` and it will fix this style issue automatically

---

"Terraform fmt" has been applied to the code successfully.